kops/kops.go

```diff
@@ -1,57 +1,30 @@
 package kops
 
 import (
+	"bytes"
 	"encoding/json"
-	"errors"
 	"fmt"
-	"io/ioutil"
-	"os"
-	"strings"
-
 	clusteroperatorv1alpha1 "github.com/infobloxopen/cluster-operator/pkg/apis/clusteroperator/v1alpha1"
 	"github.com/infobloxopen/cluster-operator/utils"
+	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"strings"
 )
 
 type KopsCmd struct {
-	devMode   bool
-	publicKey string
-	envs      [][]string
```

Comment:
Currently in master, this is hanging useless since the binary approach has taken over (I forgot to take it out) so feel free to kill it when you merge master in and stick to a function that sets the variables at runtime of the command